src/freenet/clients/http/UserAlertsToadlet.java

```diff
@@ -47,7 +50,16 @@ public void handleMethodPOST(URI uri, HTTPRequest request, ToadletContext ctx) t
 			int userAlertHashCode = request.getIntPart("disable", -1);
 			ctx.getAlertManager().dismissAlert(userAlertHashCode);
 		}
-		headers.put("Location", ".");
+		
+		
+		String redirect;
+		try {
+			redirect = request.getPartAsStringThrowing("redirectToAfterDisable", 1024);
+		} catch (SizeLimitExceededException | NoSuchElementException e) {
+			redirect = ".";
+		}
+		headers.put("Location", redirect);
```

Comment:
I'm sorry I would not have expected that browsers would redirect the user to a different domain and NOT display the different domain as the location. Why on earth do they do that?

Will fix it tomorrow.